mu-plugins/10up-plugin/src/Assets.php

```diff
@@ -0,0 +1,273 @@
+<?php
+/**
+ * Assets module.
+ *
+ * @package TenUpPlugin
+ */
+
+namespace TenUpPlugin;
+
+use TenupFramework\Module;
+use TenupFramework\ModuleInterface;
+
+/**
+ * Assets module.
+ *
+ * @package TenUpPlugin
+ */
+class Assets implements ModuleInterface {
+
+	use Module;
+
+	/**
+	 * Can this module be registered?
+	 *
+	 * @return bool
+	 */
+	public function can_register() {
+		return true;
+	}
+
+	/**
+	 * Register any hooks and filters.
+	 *
+	 * @return void
+	 */
+	public function register() {
```

Comment:
I would love to see us automate enqueueing scripts/styles. we can find all the scripts/styles in the dist folder and should be able to infer their context (or add it easily)

Reply:
This would be an amazing enhancement. CC @fabiankaegy Toby gave me some thoughts on this in Slack too.